enterprise/frontend/src/embedding-sdk/components/public/InteractiveQuestion/components/Summarize.tsx

```diff
@@ -69,26 +70,28 @@ const SummarizeInner = ({
   });
 
   return (
-    <Stack w="100%" h="100%">
-      <SummarizeAggregationItemList
-        query={query}
-        stageIndex={stageIndex}
-        aggregations={aggregations}
-        onAddAggregations={handleAddAggregations}
-        onUpdateAggregation={handleUpdateAggregation}
-        onRemoveAggregation={handleRemoveAggregation}
-      />
-      <Divider my="lg" />
-      {hasAggregations && (
-        <SummarizeBreakoutColumnList
+    <Stack className={CS.overflowHidden} h="inherit" w="100%">
```

Comment:
Not sure what happened here with the diff. Essentially I added an inner `Stack` that holds the `SummarizeAggregationItemList` and the `SummarizeBreakoutColumnList`, and acts as the scrollable area for the elements inside. This will ensure that the `Apply` and `Close` buttons stay at the bottom, while the main summarize area is scrollable